realhf/impl/model/conversion/hf_registry.py

```diff
@@ -289,7 +350,9 @@ def save(
         metadata_t = t1 - tik
         gather_cpu_t = t2 - t1
         dump_t = t3 - t2
-        logger.info(
-            f"Saving to HuggingFace Model metadata cost={metadata_t:.2f}s, gather/cpu copy cost={gather_cpu_t:.2f}s, "
-            f"dump cost={dump_t:.2f}s"
-        )
+        if os.getenv("REAL_LOG_SAVE_TIME", None) == "1":
+            logger.info(
+                f"Saving to HuggingFace Model metadata cost={metadata_t:.2f}s, "
+                f"gather/cpu copy cost={gather_cpu_t:.2f}s, "
+                f"dump cost={dump_t:.2f}s"
+            )
```

Comment:
Is this env var only used here? Maybe set log level.

Reply:
Changed to `logger.debug`.